common/scala/src/main/scala/whisk/common/Config.scala

```diff
@@ -146,11 +145,9 @@ object Config {
    * @param required a key-value map where the keys are required properties
    * @param properties a set of properties to check
    */
-  def validateProperties(required: Map[String, String], properties: Map[String, String])(
-    implicit logging: Logging): Boolean = {
+  def validateProperties(required: Map[String, String], properties: Map[String, String]): Boolean = {
     required.keys.forall { key =>
       val value = properties(key)
-      if (value == null) logging.error(this, s"required property $key still not set")
```

Comment:
why removing the loggers?
this is particularly useful when a deployment fails and you dont know why.